LEAF_Nexus/api/controllers/Group/GET.php

```diff
@@ -47,7 +47,7 @@
         $offset = (int)$_GET['offset'];
     }
 
-    return $group->listGroupEmployeesDetailed($args[0], $searchText, $offset, $limit);
+    //return $group->listGroupEmployeesDetailed($args[0], $searchText, $offset, $limit);
```

Comment:
Is there a reason to comment this line out?

Reply:
I'm not sure why I commented this out. I uncommented it.